common/script/fns/index.js

```diff
@@ -20,6 +21,7 @@ module.exports = {
   randomDrop,
   autoAllocate,
   updateStats,
+  statsComputed,
```

Comment:
I removed that I'm getting regressions like ```  4) shared.ops.scoreTask scores dailys up:
     AssertionError: expected NaN to be above 0
      at expectGainedPoints (scoreTask.test.js:37:37)
      at Context.<anonymous> (scoreTask.test.js:172:9)
```
So I'll take a look in depth on this in a bit.